public/app/features/explore/Logs/Logs.tsx

```diff
@@ -368,6 +368,11 @@ class UnthemedLogs extends PureComponent<Props, State> {
     const baseUrl = /.*(?=\/explore)/.exec(`${window.location.href}`)![0];
     const url = urlUtil.renderUrl(`${baseUrl}/explore`, { left: serializedState });
     await createAndCopyShortLink(url);
+
+    reportInteraction('grafana_explore_logs_permalink_clicked', {
+      datasourceType: row.datasourceType ?? 'unknown',
+      logRowUid: row.uid,
```

Comment:
Just brainstorming. Might it be useful to collect something about the logged line? Maybe level? (e.g. are they sharing more errors, or warnings, or just any log?).

Reply:
Good idea! Added in f0131da26ee2286145c182d54321f111e8825090.